### PR TITLE
button feature implemented

### PR DESCRIPTION
**Pull request by bharatjain7723**
#### Button

- Primary
- Link
- Icon Button
- Floating Action Button

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  line-height: 1.5;
+  color: rgb(var(--color-light));
+  text-align: center;
+  text-decoration: none;
+  vertical-align: middle;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  user-select: none;
+  background-color: transparent;
```
Comment (anupkgurung) [inline Components/Button/button.css:13]: any specific reason for using user-select in button ?

File: Components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+.btn-warning {
+  background-color: rgb(var(--warning));
+  border-color: rgb(var(--warning));
+  color: rgb(var(--color-dark));
+}
+
+.btn-info {
+  background-color: rgb(var(--info));
+  border-color: rgb(var(--info));
+  color: rgb(var(--color-dark));
```
Comment (anupkgurung) [inline Components/Button/button.css:51]: (suggestion) using hex instead of rgb would save you the time.

**Review by anupkgurung — COMMENTED** [review]
Added some comments.

File: Components/Button/button.css (hunk 2 above)
Comment (bharatjain7723) [reply Components/Button/button.css:51]: I tried using hex before. But, using it in `var()` and then in `rgba()` won't work.
 Ex- `background-color: rgba(var(--primary), 0.5)` won't work

File: Components/Button/button.css (hunk 1 above)
Comment (bharatjain7723) [reply Components/Button/button.css:13]: I was following the concept from Bootstrap.

File: Components/Button/button.css (hunk 2 above)
Comment (bharatjain7723) [reply Components/Button/button.css:51]: @anupkgurung pls review again

File: utilities.css [hunk 3]
```diff
@@ -7,4 +7,9 @@
   --danger: 220, 53, 69; /* #dc3545 */
   --background-alpha: 0.5;
   --border-alpha: 0.25;
+  --color-light: 255, 255, 255;
+  --color-dark: 0, 0, 0;
```
Comment (hiteshverma27) [inline utilities.css:11]: Try to use hex codes

File: Components/Button/button.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+.btn-primary .btn-svg-icons {
+  color: var(--color-dark);
+}
+
+.btn-floating {
+  width: 3rem;
+  height: 3rem;
+  border-radius: 50%;
+  border-width: 1px; /* thicker border distorts circle shape */
```
Comment (hiteshverma27) [inline Components/Button/button.css:81]: try to give fixed position for floating button for example compose button in gmail app

File: Components/Button/button.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+.btn-primary {
+  background-color: rgb(var(--primary));
+  border-color: rgb(var(--primary));
+}
+
+.btn-secondary {
+  background-color: rgb(var(--secondary));
+  border-color: rgb(var(--secondary));
+}
+
```
Comment (hiteshverma27) [inline Components/Button/button.css:31]: suggestion - addition of hover effect and disabled button would be a nice touch

**Review by hiteshverma27 — APPROVED** [review]
Rest looks good

**Review by anupkgurung — APPROVED** [review]
looks good.

**Pull request merged**